index.html

```diff
@@ -0,0 +1,162 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <title>Portfolio mobile first</title>
+  </head>
+  <body>
+    <main class="container">
+      <!-- =====TONIC-PAGE-==== -->
+      <section class="tonic1">
+        <div class="tonic-container">
+          <div class="image-tonic">
+            <img class="img-tanne" src="./images/tanne.svg" alt="" />
+          </div>
+          <div>
+            <h1 class="tonic-title">Tonic</hi>
+          </div>
+          <div class="counter">
+            <span>CANOPY</span>
+            <ul>
+              <li>
+                <img class="img-circle" src="./images/Counter.svg" alt="" /> Back
+                End Dev
+              </li>
+              <li>
+                <img class="img-circle" src="./images/Counter.svg" alt="" />2015
+              </li>
+            </ul>
+          </div>
+          <div class="tonic-para">
+            <p>
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required.
+            </p>
+          </div>
+          <div class="list-skills">
+            <ul>
+              <li>html</li>
+              <li>css</li>
+              <li>javaScript</li>
+            </ul>
+          </div>
+          <button type="button" class="see-pro">See Project</button>
```

Comment:
Your project button looks a bit different from the design😉, I suggest you make the background-color to be white so it can match the given design🤗

your button
<img width="242" alt="your_button" src="https://user-images.githubusercontent.com/64643995/213450793-2bdd5b9e-05fd-4338-a6ec-1b26537ce30b.png">
design button
<img width="237" alt="design_button" src="https://user-images.githubusercontent.com/64643995/213450854-37c1d942-f816-4143-bac8-7adb8dcd8ee7.png">

Reply:
implemented